apps/cowswap-frontend/src/modules/injectedWidget/updaters/InjectedWidgetUpdater.tsx

```diff
@@ -34,6 +37,12 @@ const cacheMessages = (event: MessageEvent) => {
   messagesCache[method] = event.data
 }
 
+const paramsWithoutPartnerFee = (params: CowSwapWidgetParams) => {
+  const { partnerFee: _, ...rest } = params
```

Comment:
Just curious, I've never seen this notation before to ignore the param and shut up the LINTER.

It it common? I guess is to avoid `: _partnerFee` which is longer

Reply:
Leandro suggested it: https://github.com/cowprotocol/cowswap/pull/3968#discussion_r1511248615

I don't use it very often, but I find it neat

Reply:
Defined in eslint rules https://github.com/cowprotocol/cowswap/blob/e2df98dba7858c47ebf1ce87808c457c253a9ebe/.eslintrc-common.json#L47

Anything after `_` goes, even nothing.

Reply:
yep, i know that, just I never saw in JS a var called just underscore (unless is the underscore library).

Reply:
I prefer the deconstruction method rather than copy and delete, but we can do that way too, I'm not opposed.